### PR TITLE
fix: style links to make them more accessible

### PR DESCRIPTION
**Pull request by mancristiana**
## Description
This PR changes the colour of links throughout the Help Center to distinguish links from text and improve A11Y. 

Changes overview:
* base links have a visited colour `#9358B0` 
* body links (in articles or stand-alone) always underlined and have a hover / active / focus colour `#0F3554`

## References
* https://zendesk.atlassian.net/browse/PRODA11Y-160 changes link style of "Submit a request" link and overlaps with PRODA11Y-243
* https://zendesk.atlassian.net/browse/PRODA11Y-217 changes link styles in breadcrumbs
* https://zendesk.atlassian.net/browse/PRODA11Y-243 changes link styles in the article body and article sidebar
* https://zendesk.atlassian.net/browse/PRODA11Y-244 changes link style of "sign in" link in the footer
* https://zendesk.atlassian.net/browse/PRODA11Y-267 changes link styles on landing page: promoted articles, "Join the conversation", recent activity, "See more"
### Breadcrumbs (PRODA11Y-217)
Before
z3ncriss zendesk com_hc_en-us_search_utf8=%E2%9C%93 query=general(iPad Mini) (1)

After
z3ncriss zendesk com_hc_en-us_search_utf8=%E2%9C%93 query=general(iPad Mini)

### Article (PRODA11Y-243, PRODA11Y-160)
Before
z3ncriss zendesk com_hc_en-us_articles_4418305051034-How-do-I-publish-my-content-in-other-languages- (2)

After
z3ncriss zendesk com_hc_en-us_articles_4418305051034-How-do-I-publish-my-content-in-other-languages- (3)

### Footer (PRODA11Y-160, PRODA11Y-244)
Before
Screenshot 2022-02-09 at 12 35 22

After
Screenshot 2022-02-09 at 12 34 47

### Landing page (PRODA11Y-267)
Before
z3ncriss zendesk com_hc_en-us(iPad Mini) (1)

After
z3ncriss zendesk com_hc_en-us(iPad Mini)

## Checklist

- [ ] :green_book: all commit messages follow the [conventional commits](https://conventionalcommits.org/) standard
- [ ] :nail_care: SASS files are compiled
- [ ] :arrow_left: changes are compatible with RTL direction
- [ ] :wheelchair: changes are accessible
- [ ] :memo: changes are tested in Chrome, Firefox, Safari and Edge
- [ ] :iphone: changes are responsive and tested in mobile
- [ ] :+1: PR is approved by @zendesk/vikings

### CODE REVIEW

File: styles/_variables.scss [hunk 1]
```diff
@@ -1,12 +1,6 @@
 // Variables
 
 // Colors
-$brand_color: #17494d !default;
-$brand_text_color: #fff !default; //Color for text with an accent color background
-$text_color: #333 !default;
-$link_color: #17494d !default;
-$background_color: #fff !default;
```
Comment (JakubKopys) [inline styles/_variables.scss:8]: Looks like we can't really remove those? At least not all of those, ie `$brand_color` is used in the same file, in the line `10`. Are those removals intentional?

Comment (mancristiana) [reply styles/_variables.scss:8]: These are already defined in the manifest.json file. Once we compile the SCSS those variables become available

File: manifest.json [hunk 2]
```diff
@@ -36,12 +36,19 @@
           "label": "link_color_label",
           "value": "#1F73B7"
         },
+        {
+          "identifier": "hover_link_color",
+          "type": "color",
+          "description": "hover_link_color_description",
+          "label": "hover_link_color_label",
```
Comment (luis-almeida) [inline manifest.json:43]: We don't have these strings.
If we're adding a new setting it needs to be localized first.

Comment (mancristiana) [reply manifest.json:43]: Isn't this only used for SCSS? I don't understand what needs localization?

Comment (luis-almeida) [reply manifest.json:43]: This variables are exposed as settings in the UI and need to be localized:

Screenshot 2022-02-09 at 14 35 10

They're not just available to CSS. Please see [documentation](https://github.com/zendesk/copenhagen_theme#manifest-file) for more info.

**Review by luis-almeida — APPROVED** [review]

**Pull request merged**